### PR TITLE
Added maven support

### PR DESCRIPTION
**Pull request by dartartem**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+                <artifactId>spring-boot-maven-plugin</artifactId>
+                <version>1.2.8.RELEASE</version>
+            </plugin>
+        </plugins>
+    </build>
+
+    <repositories>
+        <repository>
+            <id>eventuate</id>
+            <url>https://dl.bintray.com/eventuateio-oss/eventuate-maven-release</url>
```
Comment (cer) [inline pom.xml]: Please use jcenter repository

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        <springBootVersion>1.4.5.RELEASE</springBootVersion>
+    </properties>
+
+    <modules>
+        <module>eventuate-tram-examples-common</module>
+        <module>eventuate-tram-examples-in-memory</module>
+        <module>eventuate-tram-examples-jdbc-kafka</module>
+    </modules>
+
+    <dependencyManagement>
```
Comment (cer) [inline pom.xml]: Remove this

File: eventuate-tram-examples-in-memory/pom.xml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+        <artifactId>eventuate-tram-core-examples-basic</artifactId>
+        <version>0.1.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>eventuate-tram-examples-in-memory</artifactId>
+    <packaging>jar</packaging>
+    <name>eventuate-tram-examples-in-memory</name>
+
+    <dependencies>
+        <dependency>
```
Comment (cer) [inline eventuate-tram-examples-in-memory/pom.xml:14]: strictly speaking these dependencies should be <scope>test</scope>

File: eventuate-tram-examples-jdbc-kafka/pom.xml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+        <groupId>io.eventuate.tram.examples.basic</groupId>
+        <artifactId>eventuate-tram-core-examples-basic</artifactId>
+        <version>0.1.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>eventuate-tram-examples-jdbc-kafka</artifactId>
+    <packaging>jar</packaging>
+    <name>eventuate-tram-examples-jdbc-kafka</name>
+
+    <dependencies>
```
Comment (cer) [inline eventuate-tram-examples-jdbc-kafka/pom.xml:13]: strictly speaking these dependencies should be <scope>test</scope>

File: pom.xml [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+    <version>0.1.0-SNAPSHOT</version>
+    <packaging>pom</packaging>
+
+    <name>eventuate-tram-core-examples-basic</name>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <java.version>1.8</java.version>
+        <eventuateTramVersion>0.9.0.RELEASE</eventuateTramVersion>
+        <springBootVersion>1.4.5.RELEASE</springBootVersion>
```
Comment (cer) [inline pom.xml]: Use latest Spring Boot 2 version

**Review by cer — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (dartartem) [reply pom.xml]: ok

File: pom.xml (hunk 2 above)
Comment (dartartem) [reply pom.xml]: ok

File: eventuate-tram-examples-in-memory/pom.xml (hunk 3 above)
Comment (dartartem) [reply eventuate-tram-examples-in-memory/pom.xml:14]: ok

File: eventuate-tram-examples-jdbc-kafka/pom.xml (hunk 4 above)
Comment (dartartem) [reply eventuate-tram-examples-jdbc-kafka/pom.xml:13]: ok

File: pom.xml (hunk 5 above)
Comment (dartartem) [reply pom.xml]: ok

**Pull request merged**